tests/test_utils.py

```diff
@@ -31,8 +31,11 @@
     "llama2_7b": "/tmp/test-artifacts/llama2-7b-torchtune.pt",
 }
 
+# Inherit from tokenizer class to reuse its tokenize_messages method
```

Comment:
did you just say inherit 👀

Reply:
Best way to reduce copy-pasted code 😉. But for this case I do think it's the right thing, since it still allows us to call `tokenize_messages` (e.g. when testing on datasets) with the stripped-down tokenizer but using equivalent logic. The only usage currently is in test_slimorca.py, but I do kinda like that test, so keeping it like this for now.